packages/big-design/src/components/Timepicker/Timepicker.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { forwardRef, memo, Ref } from 'react';
+
+import { Select } from '../Select';
+
+interface Props {
+  error?: React.ReactNode;
+  label?: string;
+  onTimeChange(date: string): void;
+}
+
+interface PrivateProps {
+  forwardedRef: Ref<HTMLInputElement>;
+}
+
+export type TimepickerProps = Props & React.InputHTMLAttributes<HTMLInputElement>;
+
+function getTimeIntervals() {
+  const times = ['12:00 AM'];
+
+  for (let i = 1; i < 24; i++) {
+    i < 12 ? times.push(`${i}:00 AM`) : times.push(`${i - 12 === 0 ? i : i - 12}:00 PM`);
```

Comment:
🍹 just a nit, but why not `{i === 12 ? i : i-12}` instead of `i - 12 === 0`?

Reply:
Thanks for the feedback. Now I look at it, not sure what it was done in the first place. Updated with the suggestion. Thanks.